Assets/Scripts/Game/Tasks/GameTask.cs

```diff
@@ -158,5 +160,21 @@ protected void DestroyTask()
             
             Destroy(gameObject);
         }
+
+        /// <summary>
+        /// Adds a nav marker to the linkedGameObjects and attaches it to the task
+        /// </summary>
+        /// <param name="marker">Reference to the marker Prefab. Needs to be set in Editor per TaskFactory!</param>
+        public void attachMarker(AltMarker marker)
```

Comment:
```suggestion
        public void AttachMarker(AltMarker marker)
```
Methods should start with a capital letter